Markdown/Markdown.sublime-syntax

```diff
@@ -319,6 +320,9 @@ contexts:
         Markdown will convert this for us. We match it so that the
                         HTML grammar will not mark it up as invalid.
       scope: meta.other.valid-ampersand.markdown
+  ligatures:
+    - match: '<=>'          # <=>
+    - match: '<[-=<]'       # <- <= <<
```

Comment:
This context doesn't intend to match all possible ligatures. It instead tries to handle the edge case caused by the html tag matching or the `valid-bracket` rule. I could also try to add some lookaheads or things like that to the corresponding rules to avoid matching the tokens, but I found it a bit more complicated due to the several levels of lookaheads etc.

I found this the most secure way to fix the issue - maybe not the most elegant one.

Another edge case is the `**=` operator which is not recognized as ligature, but as the `**` is used to open bold regions, I am worried about breaking things by matching it.

Reply:
Okay, that sounds like a good approach for now, let's try it.